### PR TITLE
Fix translations displayed in Shortcuts app for list of actions

### PR DESCRIPTION
**Pull request by GianniCarlo**
## Bugfix

- The app intents were not being translated inside the Shortcuts app when showing the list of available actions

## Related tasks

#1057 

## Approach

- Remove the table parameter from the title's initialization

## Things to be aware of / Things to focus on

- I'm including a change related to the previous PR, it's just grouping the conditionals under a single function

## Screenshots

IMG_9A104298B5D5-1
IMG_BB00940A610B-1

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -22,5 +22,5 @@ jobs:
     - name: Resolve dependencies
       run: xcodebuild -resolvePackageDependencies
     - name: Build and Run tests
-      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0.1'
+      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.0'
```
Comment (SimonHaas) [inline .github/workflows/ci.yml:25]:
```suggestion
      run: xcodebuild -scheme BookPlayer test -testPlan Unit\ Tests -destination 'platform=iOS Simulator,name=iPhone 15,OS=17.2'
```
https://github.com/TortugaPower/BookPlayer/actions/runs/7262525331/job/19785852622#step:7:48

This change breaks the build. there is no OS=17.0

Comment (GianniCarlo) [reply .github/workflows/ci.yml:25]: yeah I have to circle back to this, in the original build the tests never kicked off for some reason [[job]](https://github.com/TortugaPower/BookPlayer/actions/runs/7249398846/job/19767546775), so I was testing with [macos13-arm64](https://github.com/actions/runner-images/blob/main/images/macos/macos-13-arm64-Readme.md) but the job never started for that, so I'm guessing it's only for some paid plan. I'll test out with 17.2 and hopefully the tests start without problems

Comment (GianniCarlo) [reply .github/workflows/ci.yml]: yup that did it, thanks @SimonHaas

**Comment by SimonHaas** [comment]
@GianniCarlo Thanks for your quick action!

**Comment by SimonHaas** [comment]
@GianniCarlo As seen in your first screenshot, some strings are still in english. Maybe they are not available / translated to german yet? I will look into it myself the next days.

**Comment by GianniCarlo** [comment]
@SimonHaas ah yes, those are the old shortcuts from SiriKit (they don't have a translation, I think I never figured out the proper way to translate them as they were defined in a separate special type of file for Siri intents),but they are being replaced by AppIntents, and once we drop support for iOS 14 and iOS 15, we can safely delete them from the project

**Pull request merged**